### PR TITLE
chore: rename ingest_manager endpoints to fleet

### LINKED ISSUE #342: [Fleet] update apis due to ingest_magement -> fleet api change
we need to track a bit of work here, this is just a place-holder.  lets fill it in and / or just push up the changes asap.  based on info here:
https://github.com/elastic/kibana/pull/79193

### PR DESCRIPTION
**Pull request by mdelapenya**
Please do not merge until Kibana 8.0.0-SNAPSHOT contains https://github.com/elastic/kibana/pull/79193

## What does this PR do?
It renames all HTTP endpoints from "/ingest_manager" to "/fleet/"

## Why is it important?
This PR is preparing the code for upcoming changes on Kibana side.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests for the CLI, and they are passing locally
- [x] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## Related issues

- Relates https://github.com/elastic/kibana/pull/79193
- Closes #342

## Follow-ups
As described in the Kibana PR, this must not be backported to 7.9.x

### CODE REVIEW

**Review by EricDavisX — APPROVED** [review]
looks right to me - so glad they are all encapsulated here, nicely set up for easy maintenance!

File: e2e/_suites/ingest-manager/fleet.go [hunk 1]
```diff
@@ -18,13 +18,13 @@ import (
...
-const fleetAgentsUnEnrollURL = kibanaBaseURL + "/api/ingest_manager/fleet/agents/%s/unenroll"
-const fleetEnrollmentTokenURL = kibanaBaseURL + "/api/ingest_manager/fleet/enrollment-api-keys"
-const fleetSetupURL = kibanaBaseURL + "/api/ingest_manager/fleet/setup"
-const ingestManagerAgentPoliciesURL = kibanaBaseURL + "/api/ingest_manager/agent_policies"
-const ingestManagerDataStreamsURL = kibanaBaseURL + "/api/ingest_manager/data_streams"
+const fleetAgentsURL = kibanaBaseURL + "/api/fleet/agents"
+const fleetAgentEventsURL = kibanaBaseURL + "/api/fleet/agents/%s/events"
+const fleetAgentsUnEnrollURL = kibanaBaseURL + "/api/fleet/agents/%s/unenroll"
+const fleetEnrollmentTokenURL = kibanaBaseURL + "/api/fleet/enrollment-api-keys"
+const fleetSetupURL = kibanaBaseURL + "/api/fleet/setup"
```
Comment (nchaulet) [inline e2e/_suites/ingest-manager/fleet.go:25]:
```suggestion
const fleetSetupURL = kibanaBaseURL + "/api/fleet/agents/setup"
```

**Review by cachedout — APPROVED** [review]

**Comment by EricDavisX** [comment]
the Kibana side PR is merged, fyi

**Review by nchaulet — APPROVED** [review]

**Comment by EricDavisX** [comment]
the 8.0 / master unified build failed... not sure if a new Kibana container was created or not - we may want to wait to merge, not sure.

**Comment by mdelapenya** [comment]
Merging, as I verified that latest Kibana snapshot contains the changes. Thanks!

**Pull request merged**